index.bs

```diff
@@ -4807,6 +4807,11 @@ understand the characteristics of the [=authenticators=] that they trust, based
 
 Attestation is most commonly provided during credential creation. However, if supported by the authenticator and requested by the [=[RP]=] using the {{PublicKeyCredentialRequestOptions/attestation}} parameter, attestation MAY be provided in [=assertions=].
 
+Attestations in [=assertions=] could be helpful in at least the following situations:
+
+1. For [=multi-device credentials=], the [=generating authenticator=] may have returned a meaningfully different attestation then the authenticator currently exercising the credential. Thus returning an attestation for each use of the credential allows the [=[RP]=] to observe these changes.
```

Comment:
s/then/than ?

Reply:
commented in the commit, resolving here.

Reply:
Thanks! Fixed.